pack/df.json

```diff
@@ -59,7 +59,7 @@
         "deck_limit": 3,
         "faction_code": "anarch",
         "faction_cost": 3,
-        "flavor": "\"Net space is an abstraction, a white lie protecting fragile comprehensions. Do not limit yourself.\" -z\\h/r",
+        "flavor": "\"Net <strong>space</strong> is an abstraction, a white lie protecting fragile comprehensions. Do not limit yourself.\" -z\\h/r",
```

Comment:
Won't using `<strong>` here bold the text, rather than un-italicise it like on the card image?

For example, check QPM:

https://github.com/Alsciende/netrunner-cards-json/blob/master/pack/dad.json#L114

Reply:
The few references I could quickly find for unitalicized words in flavor text suggested they were rendered using `<strong>` in the card data. This obviously isn't a perfect translation, but in the absence of counterexamples, that's what I went with.

That said, if there is precedent for just unitalicizing those emphasized words in the card data, by all means, that makes more sense.

Reply:
What's a good alternative to `<strong>` in flavor? `<em>`?

Reply:
If we use `<em>`, will it properly display as unitalicized in the italicized flavor text? Is the NetrunnerDB CSS set up to handle that?

Because if it is, then totally, `<em>` makes sense. It also makes semantic sense.

Reply:
Yeah, I thought I'd already done at least one FT like this, but my understanding is that `<em>` would have the desired effect - I can test on my local later if someone else doesn't beat me to it.